plugins/generator-1.18.2/forge-1.18.2/itemextension.definition.yaml

```diff
@@ -0,0 +1,15 @@
+templates:
+  - template: itemextension.java.ftl

```

Comment:
This template and class could be renamed to something like [name]DispenseBehavior, and should be deleted when the condition is false

Reply:
Currently, it only contains the code for the dispenser behaviour features, but in the future, we may add more features to it.

Reply:
I think it'd be cleaner to keep unrelated behaviors in different classes, similar to what's happening with composting and dispense behavior